src/test/regress/expected/upgrade_columnar_after.out

```diff
@@ -0,0 +1,134 @@
+SHOW server_version \gset
+SELECT substring(:'server_version', '\d+')::int > 12 AS server_version_above_eleven
+\gset
+\if :server_version_above_eleven
+\else
+\q
+\endif
+SET search_path TO upgrade_columnar, public, pg_catalog;
+-- test we retained data
+SELECT * FROM test_retains_data ORDER BY a;
+ a |   b   |    c    |      d
+---------------------------------------------------------------------
+ 1 | abc   | (1,4)   | {1,2,3,4}
+ 2 | pi    | (3,192) | {3,1,4,1,5}
+ 3 | earth | (4,22)  | {1,2,7,5,6}
+(3 rows)
+
+SELECT count(*) FROM test_retains_data;
+ count
+---------------------------------------------------------------------
+     3
+(1 row)
+
+SELECT a,c FROM test_retains_data ORDER BY a;
+ a |    c
+---------------------------------------------------------------------
+ 1 | (1,4)
+ 2 | (3,192)
+ 3 | (4,22)
+(3 rows)
+
+SELECT b,d FROM test_retains_data ORDER BY a;
+   b   |      d
+---------------------------------------------------------------------
+ abc   | {1,2,3,4}
+ pi    | {3,1,4,1,5}
+ earth | {1,2,7,5,6}
+(3 rows)
+
+SELECT * FROM test_retains_data ORDER BY a;
+ a |   b   |    c    |      d
+---------------------------------------------------------------------
+ 1 | abc   | (1,4)   | {1,2,3,4}
+ 2 | pi    | (3,192) | {3,1,4,1,5}
+ 3 | earth | (4,22)  | {1,2,7,5,6}
+(3 rows)
```

Comment:
I added column subsets since while developing cstore_fdw there were moments that `select count(*)` and `select *` worked but selecting some columns didn't. I don't remember the reason, but I thought it wouldn't hurt.